src/zookeeper_config.py

```diff
@@ -20,6 +20,26 @@
     f"-Djava.security.auth.login.config={ZOOKEEPER_AUTH_CONFIG_PATH}",
 ]
 
+ZOOKEEPER_PROPERTIES = """
+clientPort=2181
+dataDir=/var/snap/kafka/common/data
+dataLogDir=/var/snap/kafka/common/log
+dynamicConfigFile=/var/snap/kafka/common/zookeeper-dynamic.properties
+maxClientCnxns=60
+minSessionTimeout=4000
+maxSessionTimeout=40000
+autopurge.snapRetainCount=3
+autopurge.purgeInterval=0
+reconfigEnabled=true
+standaloneEnabled=false
+4lw.commands.whitelist=*
```

Comment:
can we please explicitly mention the needed commands?
```suggestion
4lw.commands.whitelist=mntr
```